app/src/main/java/org/oppia/app/profileprogress/ProfilePictureEditDialogFragment.kt

```diff
@@ -0,0 +1,46 @@
+package org.oppia.app.profileprogress
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.view.ContextThemeWrapper
+import androidx.fragment.app.DialogFragment
+import org.oppia.app.R
+
+/**
+ * DialogFragment that gives option to either view the profile picture or change the current profile picture.
```

Comment:
This can be a single line comment like this:

`[DialogFragment] that gives option to either view the profile picture or change the current profile picture.`